doc/guides/node-postmortem-support.md

```diff
@@ -0,0 +1,29 @@
+# Postmortem Support
+
+## Debug symbols
+
+Debug symbols are constants present in the final build which can be used by
+debuggers and other tools to navigate internal structures of the software when
+analyzing its memory (either on a running process or a core dump). Node
+provides debug symbols in its builds for V8 and Node internal structures.
+
+## V8 Debug Symbols
+
+V8 prefixes all debug symbols with `v8dbg_`, and they allow inspection of
+objects on the heap as well as object properties and references. V8 generates
+those symbols with a script (`deps/v8/tools/gen-postmortem-metadata.py`), and
+Node always includes these symbols in the final build.
+
+## Node Debug Symbols
+
+Node prefixes all debug symbols with `nodedbg_`, and they complement V8 symbols
+by providing ways to inspect Node-specific structures, like `node::Environment`,
+`node::AsyncWrap` and its descendants, objects from `utils.h` and others. Those
+symbols are provided by the script `tools/gen-postmortem-metadata.py`.
+
+## Tools and References
+
+* [https://github.com/nodejs/llnode](llnode): LLDB plugin
+* [https://github.com/joynet/mdb_v8](mdb_v8): mdb plugin
```

Comment:
Linter failed because `remark-lint-prohibited-strings` does not skip "v8" here, cc @Trott 

BTW links here are broken, I think you are going for things like `[llnode](https://github.com/nodejs/llnode)`?

Reply:
Link on line 27 should contain `joyent` rather than `joyent`.

Reply:
@joyeecheung The `mdb_v8` in the URL should be ignored by the linter. The `mdb_v8` meant for the link text will be fine if it is wrapped in backticks.